pmm-tests/check_upgrade.py

```diff
@@ -60,6 +61,11 @@ def main():
         verify_command(f"docker exec {pmm_server_docker_container} rpm -qa | grep percona-qan-api2-{args.version}")
         verify_command(
             f"docker exec {pmm_server_docker_container} rpm -qa | grep percona-dashboards-{args.version}")
+
+        ### PMM-12223 - Verify Clickhouse  is v22.8 or later since 2.41.0
```

Comment:
look like u mean v23.8 in the comment ?

Reply:
Actually, no - it's a copy of the Ticket's title: see https://jira.percona.com/browse/PMM-12223 
But I suppose you're right and I will change the it be more accurate in the sibling PR: https://github.com/percona/pmm-qa/pull/645